gui/theme_model.py

```diff
@@ -174,6 +174,29 @@ def value_filter(colorscheme):
 
     {
         'type': 'separator',
+        'display_name': 'Caret'
```

Comment:
please change this to something like "Text input caret" or in other way specifying what's that, it can be confusing for new users